app/src/main/java/com/duckduckgo/app/survey/ui/SurveyActivity.kt

```diff
@@ -165,5 +165,10 @@ class SurveyActivity : DuckDuckGoActivity() {
                 viewModel.onSurveyFailedToLoad()
             }
         }
+
+        override fun onRenderProcessGone(view: WebView?, detail: RenderProcessGoneDetail?): Boolean {
+            viewModel.onSurveyFailedToLoad()
```

Comment:
Is this valid? It eventually leads to a call to `webView.gone()` however according to the docs:

> The given WebView can't be used, and should be removed from the view hierarchy, all references to it should be cleaned up

In the case of an `onRenderProcessGone` failure the `WebView` should removed and set to null not hidden.

Reply:
Yes, it's valid to call `webView.gone()` but not enough. I will improve this part in order to do a better clean up for that `WebView`.

Reply:
I went for adding extra logic inside `showError` in order to fully remove the `WebView`, instead of creating a new method in `SurveyViewModel` + a new `Command` to destroy the `WebView`.
Since currently when an error happens user can't interact with Survey, I just destroy the WebView too, there's no need to keep it alive.